prime-router/metadata/fhir_mapping/hl7/secondary/Destination.yml

```diff
@@ -0,0 +1,38 @@
+name:
```

Comment:
I'm curious if there was a reason destination wasn't mapped in the first pass. Totally not blocking at at all but would want to know if there is any historical knowledge on this.

Reply:
Also do we need to update the mapping spreadsheet to reflect that we've added this?